packages/@okta/vuepress-site/docs/guides/sharing-cert/overview/index.md

```diff
@@ -2,25 +2,20 @@
 title: Overview
 ---
 
-[Application key credential objects](/docs/reference/api/apps/#application-key-credential-model)
-contain an opaque key ID (`kid`) and corresponding certificate.
-To use the [API](/docs/reference/api/apps/#clone-application-key-credential)
-to share application key credentials between apps, you need to create and use a new credential in one app,
-then share and update the credential in another app.
+[Application Key Credential objects](/docs/reference/api/apps/#application-key-credential-model)
+contain an opaque key ID (`kid`) and corresponding certificate. To use the [API](/docs/reference/api/apps/#clone-application-key-credential) to share application key credentials between apps, you need to create and use a new credential in one app, and then share and update the credential in another app.
 
-Sharing certificates is useful for Okta orgs that have apps with [sign-on modes](/docs/reference/api/apps/#sign-on-modes) `SAML_2_0`, SAML 1.1, or `WS_FEDERATION`.
+### Why should I do this?
 
-### Why Should I Do This?
+When configuring multiple apps, you might need them to accept the same identity provider (IdP). In that case, the assertions from the two apps must be signed by the same key. Sharing certificates is useful for Okta orgs that have apps with [sign-on modes](/docs/reference/api/apps/#sign-on-modes) such as `SAML_2_0`, SAML 1.1, or `WS_FEDERATION`.
```

Comment:
"SAML 1.1" isn't listed in that linked list of possible sign-on modes. Can a dev confirm that it's still supported? I might suggest replacing "SAML 1.1" with "OPENID_CONNECT" as a more modern sign-on mode.